sdks/cpp/README.md

```diff
@@ -115,34 +115,49 @@ For more information, you can also read the [SDK Overview](../), check out [sdk.
 When running on Agones, the above functions should only fail under exceptional circumstances, so please 
 file a bug if it occurs.
 
-## Dynamic and Static Libraries
-
-In the [releases](https://github.com/googlecloudplatform/agones/releases) folder
-you will find two archives for download that contain both static and dynamic libraries for building your
-game server on Linux:
-
-- `argonsdk-$(VERSION)-dev-linux-arch_64.tar.gz`: This includes all the 
-headers as well as dynamic and static libraries that are needed for development and runtime.
-- `argonsdk-$(VERSION)-runtime-linux-arch_64.tar.gz`: This includes just the dynamic libraries that 
-are needed to run a binary compiled against the SDK and its dependencies.
-
-### Building the Libraries
+## Building From Source
+To build Agones C++ SDK from sources you need to set up your development environment and install following tools:
+- CMake (https://cmake.org/download/)
+- OpenSSL (https://github.com/openssl/openssl)
+- Git (https://git-scm.com/downloads)
+- gRPC (https://github.com/grpc/grpc)
```

Comment:
What does installing gRPC mean here? 

The cmake installation should be as turnkey as possible. If they user has to manually install gRPC themselves,  this defeats the purpose of the tool we're trying to make here. The user really shouldn't care what the communication protocol is.

The previous incarnation did this really well:
https://github.com/GoogleCloudPlatform/agones/pull/421/files#diff-8ffc59e90c0bca72cf7710844b68352dR28

Reply:
Cpp SDK depends on gRPC. Unfortunately, we can't simply install gRPC for C++, as [mentioned here](https://grpc.io/blog/installation). So we need to do a manual build of gRPC which depends on OpenSSL and so on.
I think we should do next steps, to build SDK and avoid OpenSSL dependency:
1) Download gRPC repo (if path to gRPC is not specified)
2) Build only `grpc_unsecure` target (gRPC cmake package scripts will not be generated)
3) Use our own find package script that will have all references to include/lib directories of gRPC
4) Build SDK
3rd step is necessary, so end user can link with SDK, gRPC and all other dependencies without hardcoding. We need to support both Debug/Release builds for IDE-s such XCode and MSVS.

@sylvainduchesne what do you think?